iac-extensions/terraform/src/main/resources/org/sonar/l10n/terraform/rules/tflint/rules.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "key": "E0000",
+    "name": "Parsing error found when parsing the template",
+    "url": "https://github.com/aws-cloudformation/cfn-lint/blob/main/docs/rules.md#rules-1",
+    "tags": [
+      "base"
+    ],
+    "description": "Checks for JSON/YAML formatting errors in your template",
+    "constantDebtMinutes": 0,
+    "type": "BUG",
+    "severity": "MAJOR"
+  },
+  {
+    "key": "tflint.fallback",
+    "name": "Cfn-lint Rule",
```

Comment:
I don't like the cfn lint reference, but i think this will be fixed with the next ticket.